device/mellanox/x86_64-mlnx_msn4410-r0/ACS-MSN4410/port_config.ini

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn4700-r0/ACS-MSN4700/port_config.ini
```

Comment:
it cannot be the same. port_config.ini should be different as the speed is different for the qsfpdd ports and qsfp28

Reply:
[ACS-MSN4700/port_config.ini](https://github.com/Azure/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/port_config.ini) doesn't contain the speed configuration - there is only naming of interfaces, lanes, aliases, and indexes